_posts/2019-01-10-our-okr-process.md

```diff
@@ -93,51 +91,52 @@ After OKR-Workshop II everyone is waiting for the official communication. The at
+So, in the end, you can look up your name in a big list employees to see what you'll be working on. Especially in the OKR-Workshops for a new year, this list is quite interesting.
 
 <img src="/img/posts/2019-01-10-our-okr-process/christmas.svg" alt="" class="filler"/>
 
-There are often big changes on the company level but also for individual resources. <em>The list</em> is usually sent a few days after the second OKR-Workshop. That means that most people receive it right before their Christmas vacation or they are already on vacation. But this has also a bright side: People often react reluctant to changes <em>at first</em> - but now they have time to reflect on those changes in their vacation. Or they can relax in their vacation and learn about the changes in the new year: So, in the end, everyone has fresh energy for those changes.
+There are often big changes on the company level but also for individual employees for the next year. People often react reluctant to changes <em>at first</em> - but now they have time to reflect on those changes in their vacation. Or they can relax in their vacation and learn about the changes in the new year: So, in the end, everyone has fresh energy for those changes.
 
 ### Deciding on a new team setup and dealing with the changes
 
-When we create <em>the list</em> of assignments we try to give every resource focus: usually every employee is assigned with 100% to one initiative, sometimes we have resources that are assigned with a 50/50 or 25/75 split to 2 projects. But we aim to give everyone focus.
+When we create <em>the list</em> of assignments we try to give every person focus: usually every employee is assigned with 100% to one initiative, sometimes we have a person that is assigned with a 50/50 or 25/75 split to 2 projects. But we aim to give everyone focus.
 
 Sometimes in this process old teams are removed because they are no longer a priority and hence some topics, services, and half-finished projects are abandoned and we need a clear process to tackle this:
 
 The way we usually do that is that we always try to launch little MVPs as fast as possible. This is to avoid working on something that is never launched because your team no longer exists or switches what you're working on. It also means that sometimes those MVPs stay live for months and years - but since the V in MVP stands for <em>viable</em> that shouldn't be a big problem.
 
 For code, projects and services that no longer have an owner, because the team that created and maintained it no longer exists, HolidayCheck has another process:
-In our Github Enterprise instance, we have one repository that contains a markdown file with abandoned services. And since the people that worked on those services and code are still in the company, management thinks they can also keep maintaining those old things.
+We have a list of every "abandoned" service that notes down who will be responsible to maintain it.
```

Comment:
Do we really have those lists, including responsible maintainers?

Reply:
Yes, I'll send you the link.